pkgs/games/pingus/default.nix

```diff
@@ -1,34 +1,19 @@
-{stdenv, fetchurl, fetchpatch, scons, SDL, SDL_image, boost, libpng, SDL_mixer
-, pkgconfig, libGLU, libGL}:
-let
-  s = # Generated upstream information
-  {
-    baseName="pingus";
-    version="0.7.6";
-    name="pingus-0.7.6";
-    hash="0q34d2k6anzqvb0mf67x85q92lfx9jr71ry13dlp47jx0x9i573m";
-    url="http://pingus.googlecode.com/files/pingus-0.7.6.tar.bz2";
-    sha256="0q34d2k6anzqvb0mf67x85q92lfx9jr71ry13dlp47jx0x9i573m";
+{stdenv, fetchgit, cmake, SDL2, SDL2_image, boost, libpng, SDL2_mixer
+, pkgconfig, libGLU, libGL, git, jsoncpp }:
+stdenv.mkDerivation rec {
+  pname = "pingus";
+  version = "unstable-0.7.6.0.20191104";
+
+  nativeBuildInputs = [ pkgconfig git ];
+  buildInputs = [ cmake SDL2 SDL2_image boost libpng SDL2_mixer libGLU libGL jsoncpp ];
```

Comment:
`cmake` should be in `nativeBuildInputs`.

Reply:
this should be fixed in master, as this is only a backport

Reply:
Agreed, so maybe this PR should wait until a new PR against `master` is merged.

Reply:
I've submitted one and cherry-picked it here.